flow-report/src/util.ts

```diff
@@ -48,6 +48,41 @@ export function getScreenshot(reportResult: LH.ReportResult) {
   return fullPageScreenshot || null;
 }
 
+export function computeFilename(flowResult: LH.FlowResult) {
```

Comment:
these are both mostly a duplicate of report-ui-features, right? do you see a way to share these easily?

Reply:
Ya probably a way to do this.